app/javascript/packages/eslint-plugin/README.md

```diff
@@ -0,0 +1,38 @@
+# `@18f/eslint-plugin-identity`
```

Comment:
I feel like we prefix all our projects with `identity-`, does `eslint-plugin` come first by convention?

Reply:
> I feel like we prefix all our projects with `identity-`, does `eslint-plugin` come first by convention?

Yeah, in fact more than just convention, it's strictly necessary to be prefixed 😕  See https://github.com/eslint/eslint/issues/5442.

Also mentioned in original comment:

> * ESLint is rather particular about how plugins are named, so I had to break with `@18f/identity-` convention here, opting for `@18f/eslint-plugin-identity` to satisfy the `eslint-plugin-` prefix requirement.